pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/elasticexport/DeletedDataExportTest.java

```diff
@@ -59,6 +60,11 @@ public static void cleanUpAfter() {
     }
 
 
+    // TODO: remove or modify this test. We do not want to have export/migrate fixup onc history records that
+    // were not deleted properly (that is, the onc histories were not deleted from the ES).
+    // There was a one-time need for this capability, but we cannot depend on it long-term without making significant
+    // changes to the export/migrate code. -DC
+    @Ignore
```

Comment:
I think we can remove this test. This was just testing a one time functionality that we needed at the time and it has already been used.